src/utils/UpdateNotification.js

```diff
@@ -115,32 +115,6 @@ define(function (require, exports, module) {
             locale = locale.substring(0, 2);
         }
 
-        //AUTOUPDATE_PRERELEASE_BEGIN
```

Comment:
I think instead of inserting and removing code each time we do a pre-release, we can just check the `update_info_url` to see if it contains pre-release, in which case we default to 'en' otherwise go with the release method of adding URL params

Reply:
Changed URL `update_info_url` format to address the issue.